src/widgets/tabs.rs

```diff
@@ -44,6 +44,8 @@ pub struct Tabs<'a> {
     padding_left: Line<'a>,
     /// Tab Right Padding
     padding_right: Line<'a>,
+    /// Whether padding gets highlighted when selected or not
+    highlight_padding: bool,
```

Comment:
Let's use an enum for this. https://rust-lang.github.io/api-guidelines/type-safety.html#c-custom-type